document/js-api/index.bs

```diff
@@ -1103,7 +1103,23 @@ The <dfn method for="Global">type()</dfn> method steps are:
 
 </div>
 
-<h3 id="exported-function-exotic-objects">Exported Functions</h3>
+<h3 id="exported-functions">Exported Functions</h3>
+
+<pre class="idl">
+callback JSFunction = any (any... arguments);
+
+dictionary FunctionType {
+  required sequence&lt;ValueType> parameters;
+  required sequence&lt;ValueType> results;
+};
+
+[LegacyNamespace=WebAssembly, Exposed=(Window,Worker,Worklet)]
+interface Function : JSFunction {
```

Comment:
Could you suggest a better way of handling `global.Function` issue?
I haven't found any definition of JSFunction I can refer to, because there is still no IDL for full JS.
I don't expect `WebAssembly.Function` will be generated from WebIDL part(at least in V8 it wont work anyway), so I would consider it a notational compromise to let the proposal move without updating WebIDL spec.